scripts/ingests/ingest_BYW_2024.py

```diff
@@ -0,0 +1,38 @@
+from scripts.ingests.ingest_utils import *
+from scripts.ingests.utils import *
+from astropy.io.ascii import ascii
+import astropy.units as u
+from astropy.coordinates import Angle
+
+SAVE_DB = True  # save the data files in addition to modifying the .db file
+RECREATE_DB = True  # recreates the .db file from the data files
+# LOAD THE DATABASE
+db = load_simpledb('SIMPLE.db', recreatedb=RECREATE_DB)
+
+
+# live google sheet
+link = "https://docs.google.com/spreadsheets/d/1JFa8F4Ngzp3qAW8NOBurkz4bMKo9zXYeF6N1vMtqDZs/edit#gid=0"
```

Comment:
The problem is here: you need to change the end of this link to something different. It shouldn't be edit. Look at allys script to find the correct suffix. 
```suggestion
link = "https://docs.google.com/spreadsheets/d/1JFa8F4Ngzp3qAW8NOBurkz4bMKo9zXYeF6N1vMtqDZs/<something different>"
```

Reply:
Originally I had "pub?output=csv" at the end  of the link but it gave HTTP Error 404 Unauthorized, but keeping it as it's original link seemed to resolve the issue.